app/controllers/npq_separation/migration/migrations_controller.rb

```diff
@@ -2,23 +2,31 @@ class NpqSeparation::Migration::MigrationsController < ApplicationController
   before_action :require_super_admin
 
   def index
-    @data_migrations = Migration::DataMigration.all
+    @data_migrations = Migration::DataMigration.order(model: :asc, worker: :asc).all
     @in_progress_migration = @data_migrations.present? && !@data_migrations.all?(&:complete?)
     @completed_migration = @data_migrations.present? && @data_migrations.all?(&:complete?)
   end
 
   def create
-    Migration::Migrator.prepare_for_migration
+    Migration::Coordinator.prepare_for_migration
     MigrationJob.perform_later
 
     redirect_to npq_separation_migration_migrations_path
   end
 
   def download_report
-    data_migration = Migration::DataMigration.find(params[:id])
-    failures = Migration::FailureManager.new(data_migration:).all_failures
+    data_migrations = Migration::DataMigration.complete.where(model: params[:model])
```

Comment:
maybe this is something that can live in the failure manager or another service so we can test this outside the controller?

Reply:
Yeah that's fair, this was a bit of a rush change at the end, I'll clean it up